CHANGELOG.md

```diff
@@ -69,6 +68,9 @@
 - Tap outside or swipe to dismiss the bottom sheets
 - Tap outside to dismiss the dialogs
 
+### Fixes
```

Comment:
We have hot fixed 4 more commits to that release. Let's add those as well (please refer the commits from the release/2021-07-12 branch)

```
- Clean up `gradle.properties`
- Extract product flavor build config fields into `defaultConfig`
- Push records to server during sync in batches
- Allow push and pull sync batch sizes to be configured separately
```

Reply:
The sync records need to go into a new release since they were hotfixed _after_ the current release was rolled to 25%.

Reply:
I'll make that change once the changes are live.